src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformInterop.java

```diff
@@ -296,6 +296,18 @@ static PorterDuff.Mode getPorterMode(int mode) {
         }
     }
 
+    private static RequestManager safeGlideWith(Context context) {
+        // Check if context is an Activity and if it's destroyed
+        if (context instanceof Activity) {
+            Activity activity = (Activity) context;
+            if (activity.isDestroyed()) {
+                // Use application context as fallback to avoid IllegalArgumentException
+                return Glide.with(activity.getApplicationContext());
+            }
+        }
+        return Glide.with(context);
+    }
+
```

Comment:
This is a different approach than:
* https://github.com/dotnet/maui/pull/29780

Honestly, #29780 seems better to me, can we see if it solves this issue instead?